data-access/basic_data_access.ipynb

```diff
@@ -5,8 +5,31 @@
    "id": "811929f5-5aaf-4c4c-af1c-0c63bd51961e",
    "metadata": {},
    "source": [
-    "# Basic data access \n",
-    "This notebook showcases helper functions from `climakitae` that enable you to access and export the AE catalog data, while also allowing you to perform spatial subsetting and view the data options in an easy-to-use fashion. These functions could be easily implemented in a python script."
+    "# Data Access Notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "19abd63d-4e03-4634-a6a2-1fca9c967792",
+   "metadata": {},
+   "source": [
+    "In this notebook, we’ll be walking through how to retrieve climate data from `climakitae` through a series of real-world analyses. These include visualizing time-series temperature data, retrieving warming-level data, and calculating 1-in-X metrics—all designed to help you get familiar with how `climakitae` works in practice."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "46f7d3a6-1d86-4f2f-964d-3c00f46dffe8",
+   "metadata": {},
+   "source": [
+    "**Intended Application:** As a user, I want to **<span style=\"color:red\">understand how new core in `climakitae` works and how I can use it for my own climate analyses.</span>**\n"
```

Comment:
As far as I can tell, this notebook doesn't define or explain what "new core" is (as in, the actual term "new core"). Either we should have an explanation or reword the goal to focus on data access with climakitae